src/components/FilterList.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+
+export default function FilterList({ phrases, onFilterClick, filterButtons }) {
+  const allExistingNames = [...new Set(phrases.map(item => item.name))];
+  const categoryTagsAndAll = ['Alle', ...allExistingNames].sort();
+  function handleFilterCick(name) {
+    onFilterClick(name);
+  }
```

Comment:
This function does not add any functionality other than calling onFilterClick. So you can delete it and replace all handleFilterClick calls with onFliterClick

Reply:
thx good point ;)